packages/back-end/src/services/stats.ts

```diff
@@ -65,20 +65,25 @@ export interface AnalysisSettingsForStatsEngine {
   phase_length_days: number;
   alpha: number;
   max_dimensions: number;
+  decision_metric: string;
+  bandit: boolean;
+  bandit_weights_seed: number;
 }
 
 export interface BanditSettingsForStatsEngine {
-  reweight: boolean; // todo: implement in python
   var_names: string[];
   var_ids: string[];
-  decision_metric: string;
-  bandit_weights_seed: number;
   weights: { // todo: implement in python
     date: Date;
     weights?: number[];
   }[];
+  reweight: boolean; // todo: implement in python
+  decision_metric: string;
+  bandit_weights_seed: number;
+  alpha: number; 
```

Comment:
use the analysis settings alpha? (org/exp setting)

Reply:
remove this one

Reply:
now using analysis settings alpha and removed alpha from BanditSettingsForStatsEngine